src/System.Data.Odbc/src/System/Data/Odbc/odbcmetadatacollectionnames.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It'd be nice if the file name was cased correctly, e.g. OdbcMetaDataCollectionNames.cs